CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Kube-queue Release Notes
+
+## v0.1.0
+### Features
+- Build a framework to support different
+  queueing policies and quota systems
+- Support two basic queueing policies for the elements in single queue: FIFO、Priority
+- Support dynamic adjustment of job priority in queue
+- Integrate with the ResourceQuota in Kubernetes
+- Integrate with TF-operator、Pytorch-operator for distribute deeplearning training
```

Comment:
Could we change `TF-operator` and `Pytorch-operator` to `TFJob` and `PyTorchJob`? Also, change `、` to `, `

Reply:
Done